src/screens/BodyAnalysisCameraScreen.tsx

```diff
@@ -119,26 +120,25 @@ export const BodyAnalysisCameraScreen = ({ route, navigation }) => {
       setUploadStatus(null);
     }
   };
-
   if (!permission) {
-    // Camera permissions are still loading
-    return <View />;
+    return <View />
```

Comment:
It's wrong we should show Loading info or anything to the user.

Reply:
I would leave this as is. It's because this view is visible only a fraction of a second. If we will put a loder there it will always look like a glitch.